### DISCUSSION TITLE
Uninstalling

### DISCUSSION THREAD

**Discussion by RaghavKhandelwal31** [rafaelmardojai/firefox-gnome-theme#612] [Q&A]
Hi,
I tried to uninstall this theme but am not able to. I tried to follow the instructions but cant find the chrome folder. 
I even reinstalled Firefox but still the gnome theme is there. 
Can you please help me out. Thanks

**Comment by rafaelmardojai** [comment]
So the profile folder you open from `about:support` doesn't have the chrome folder?

**Reply by RaghavKhandelwal31** [reply]
Just to be clear I have to type `about:support` in Firebox search bar right? I could't find any such folder there. 
I am using Firefox 112.0.2(64 bit) in Windows 11.

**Reply by rafaelmardojai** [reply]
Yes, you should see a big table bellow, there you need to lookup for a row labeled with `Profile Directory`, there should be a `Open Directory` button.

**Reply by rafaelmardojai** [reply]
Hmm, If you're using Windows 11, how did you installed the theme?

**Reply by RaghavKhandelwal31** [reply]
I just followed your instructions written on the homepage.

**Reply by RaghavKhandelwal31** [reply]
![These are the folders that I have in the Profile Directory](https://user-images.githubusercontent.com/58084165/236735795-176122a1-5827-4c59-9606-7dbb3986bf05.png)

**Reply by rafaelmardojai** [reply]
So this is the contents of the folder you open from `about:support` right? It's really weird that no chrome folder is present.

**Reply by rafaelmardojai** [reply]
Also there should be other profiles folders along this one, check them and remove any `chrome` folder.

Other option is to set `toolkit.legacyUserProfileCustomizations.stylesheets` on `about:config`.

**Reply by RaghavKhandelwal31** [reply]
![image](https://user-images.githubusercontent.com/58084165/236741150-fcc519d8-19b9-4932-a97e-80491e4681b5.png)
So I have to set this `True', right?

**Reply by RaghavKhandelwal31** [reply]
So I did set `toolkit.legacyUserProfileCustomizations.stylesheets` to `True` and opened Firefox again but it was the same. 

I also opened profile folder's under `about:profiles` -> `Profile: default-release`. Both the `Root Directory` and `Local Directory` had no folders.

**Reply by rafaelmardojai** [reply]
>  So I have to set this `True', right?

No, it should be false. And It begin false and still showing the theme is pretty weird, this theme can't work without that preference.

**Reply by RaghavKhandelwal31** [reply]
So what can I do next? Should I delete every folder with Mozilla or Firefox in the name?

**Comment by RaghavKhandelwal31** [comment]
Hi
I installed Firefox again today, after deleting every folder folder with Mozilla or Firefox from everywhere including 'Local Roaming Program Files` . But still it loads up with the Gnome theme. What should I do now?
